### DISCUSSION TITLE
Mapping  npi with its corresponding tin details

### DISCUSSION THREAD

**Discussion by Nabins** [CMSgov/price-transparency-guide#647] [Q&A] [answered]
When we search for any procedure, it shows the doctors(NPI) and NPI address . Is there any way to link the npi with their corresponding TIN details (workplace of NPI).

**Comment by shaselton-usds** [comment, marked as answer]
The NPI and TIN should be linked to the item/service code being disclosed. This was an important factor when thinking through the initial design of the schema. If the MRF is following the schema specifications, take a look at the [`provider_groups`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-rate-details-object) attribute for the appropriate item/service code in question.
